CHANGES.md

```diff
@@ -28,6 +28,7 @@
 ## I/Os
 
 * Support for X source added (Java/Python) ([#X](https://github.com/apache/beam/issues/X)).
+* Projection Pushdown optimizer is now on by default for streaming, matching the behavior of batch pipelines since 2.38.0. If you encounter a bug with the optimizer, please file an issue and disable the optimizer using pipeline option `--experiments=disable_projection_pushdown`.
```

Comment:
Would it be worthwhile to mention something about upgrade compatibility?

Reply:
Added to the "Breaking Changes" section.